backend/integration_test/elasticsearch/count_service_test.go

```diff
@@ -504,6 +504,63 @@ func TestSpanCount(t *testing.T) {
 	})
 }
 
+func TestAlgorithm(t *testing.T) {
```

Comment:
Proof the test fails on main
![Proof that Test Fails on Main](https://github.com/user-attachments/assets/d192e25d-6ba2-489a-84c7-256f7a6e106a)